docs/evaluation/how_to_guides/evaluation/metric_type.mdx

```diff
@@ -0,0 +1,74 @@
+import {
+  CodeTabs,
+  python,
+  typescript,
+} from "@site/src/components/InstructionsWithCode";
+
+# How to return categorical vs numerical metrics
+
+:::info Key concepts
+
+- Metrics
+
+:::
+
+LangSmith supports both categorical and numerical metrics, and you can return either when writing a [custom evaluator](../../how_to_guides/evaluation/custom_evaluator).
+
+For an evaluator result to be logged as a numerical metric, it must returned as:
```

Comment:
this is only true for python, since js can't return just an int, right?

We should be clear about this